tests/_support/Page/Acceptance/Administrator/ControlPanelPage.php

```diff
@@ -0,0 +1,13 @@
+<?php
+namespace Page\Acceptance\Administrator;
+
+use Page\Acceptance\Administrator\AdminPage;
+
+class ControlPanelPage extends AdminPage
+{
+    public static $url = "/administrator/index.php";
+
+    public static $pageTitle = 'Control Panel';
+
+    public static $pageTitleContext = ['class' => 'page-title'];
```

Comment:
Imo this array should be even private / protected. And have a method `public static getLocator($title, $default = null)` in the abstract class (AdminPage or whatever).

We could also make an additional shortcut method, like we have in Joomla with for example JText::_, so you later only use:

`Login::_('pageTitle');`